config/locales/devise.fr.yml

```
@@ -40,7 +40,7 @@ fr:
         title: "Inscription"
         invitation_token: "Code d’invitation"
         submit_button: "Créer son compte"
-        click_on_link_sent: "Veuillez cliquer sur le lien dans l'email qui vient de vous être envoyé"
+        click_on_link_sent: "Veuillez cliquer sur le lien dans l'email qui vient de vous être envoyé, pour confirmer votre inscription."
```

la virgule me paraît en trop ici

```suggestion
        click_on_link_sent: "Veuillez cliquer sur le lien dans l'email qui vient de vous être envoyé pour confirmer votre inscription."
```